policies/travel-policy.md

```diff
@@ -206,9 +204,9 @@ Entitlement Table** below:
 ##### Meal Entitlement Table
  | Period | Cost |
  | --- | --- |
- | Full Day | Up to US\$200/day |
- | Partial Day | Up to US\$100/day |
- | Entertainment\* | Full Cost of meal. Alcoholic beverages up to US\$20 per person per meal |
+ | Full Day | Up to US $150 a day |
```

Comment:
It might be useful to find out the "normal official" allowance in your contry, if you want to compare.
For swedes going to Australia, it's 661 SEK (almost AUD 80, or 60 USD) / day, according to our national tax department.

Reply:
It's not so unrealistic if you check per diems set by the law per country see https://www.per-diems.info/

Reply:
Nice site.  I wonder where they get their information (for Sweden for example).
That being said, I'm not gonna protest this rate :wink: